libcontainer/cgroups/fs/cpuset.go

```diff
@@ -48,7 +50,109 @@ func (s *CpusetGroup) Remove(d *cgroupData) error {
 	return removePath(d.path("cpuset"))
 }
 
+func getNumericStat(path string, filename string, extracted *[]uint16) error {
+	fileContent, err := fscommon.GetCgroupParamString(path, filename)
+	if err != nil {
+		return err
+	}
+	if len(fileContent) == 0 {
+		return fmt.Errorf("%s found to be empty", filepath.Join(path, filename))
+	}
+
+	for _, s := range strings.Split(fileContent, ",") {
+		if strings.Contains(s, "-") {
```

Comment:
you can avoid this check and go right to splitting. If `len(splitted) == 1` it means there is no dash.

Reply:
2 or 3 could also mean no dash